cmd/meshconvert.cpp

```diff
@@ -81,6 +85,10 @@ void run ()
       case 2: transform->set_voxel2real(); break;
       case 3: transform->set_real2voxel(); break;
       case 4: transform->set_fs2real   (); break;
+      case 5:
+        transform->set_warp();
+        transform->set_warp_image(Image<float>::open(opt[0][1]));
```

Comment:
```suggestion
        transform->set_warp_image(H.get_image<float>());
```

Also `transform->set_warp()` shouldn't be necessary; it should be possible to set that internal flag when `set_warp_image()` is called.